src/Symfony/Component/DependencyInjection/ParameterBag/EnvPlaceholderParameterBag.php

```diff
@@ -62,9 +62,17 @@ public function getEnvPlaceholders()
 
     /**
      * Merges the env placeholders of another EnvPlaceholderParameterBag.
+     *
+     * @param EnvPlaceholderParameterBag $bag
```

Comment:
I added it because it looks like most of the docblocks for methods in the Symfony core either use `{@inheritdoc}` or an explicit description of parameters. You're right (and I'm sure the clean code principles would agree with you) that it doesn't add anything by looking at it, although inside an IDE like PHPStorm people will use the quick documentation feature which makes it a little bit more understandable what the type parameter type is, but of course I can remove it - just let me know